### PR TITLE
refactor(switch): Custom properties.

### PR DESCRIPTION
**Pull request by bardosmisi**
### <strong>Pull Request</strong>

Fixes the switch part of APM-266103.

#### Type of PR
Other (if none of the above apply)

### CODE REVIEW

File: libs/barista-components/switch/src/switch.scss [hunk 1]
```diff
@@ -3,7 +3,50 @@
 @import '../../core/src/style/transition';
 @import '../../core/src/theming/theming';
```
Comment (tomheller) [inline libs/barista-components/switch/src/switch.scss:4]: We can also drop this import, as we no longer need to call the `dt-apply-theme()` mixin.
```suggestion
```

File: libs/barista-components/switch/src/switch.scss [hunk 2]
```diff
@@ -3,7 +3,50 @@
...
 @import '../../style/font-mixins';
-@import './switch-theme';
+
+:host {
+  --content-color: #{$default-font-color};
+  --dt-switch-bar-background-radial: #{$gray-700};
+  --dt-switch-bar-border-color: #{$turquoise-600};
+  --dt-switch-bar-bg-border-color: transparent;
```
Comment (tomheller) [inline libs/barista-components/switch/src/switch.scss:11]: Please keep abbreviations consistent. You are sometimes using `bg` and sometimes `background`. 
I understand that some of these variables could get quite long, I would opt for `background`.
```suggestion
  --dt-switch-bar-background-border-color: transparent;
```

**Review by peter-affenzeller — APPROVED** [review]
except for the inconsistent abbreviations mentioned by tom - lgtm 🐰

File: libs/barista-components/switch/src/switch.scss (hunk 2 above)
Comment (bardosmisi) [reply libs/barista-components/switch/src/switch.scss:11]: Yes, I was thinking about that during implementation, but now I think ```background``` is better.

File: libs/barista-components/switch/src/switch.scss (hunk 1 above)
Comment (bardosmisi) [reply libs/barista-components/switch/src/switch.scss:4]: Removed, thanks 👍

**Review by tomheller — APPROVED** [review]

**Review by lukasholzer — APPROVED** [review]

**Comment by tomheller** [comment]
IN the past we have considered these features. But I'm fine with keeping it as a refactor.

**Pull request merged**